src/app/footer/footer.component.css

```diff
@@ -17,18 +17,69 @@
 .container-fluid,
 a {
   background-color: #21335b;
-  color: white;
+  color: #e8eaf6;
 }
 
 .container-fluid {
-  padding-top: 20px;
+  padding-top: 2rem;
 }
 
-.container-fluid,
 ul {
   list-style: none;
 }
 
-.footer-list li:first-child {
-  font-weight: bold;
+/* the title of each column */
+.title {
+  /* used for making the text semi-bold */
+  text-shadow: 0 0 0 #c5cae9;
+  color: #c5cae9;
+  text-transform: uppercase;
+  padding: 1rem 0;
+}
+
+.sponsor-img {
+  height: 60px;
```

Comment:
@coverbeck We don't want the image to resize according to the screen size so that's why I left it as `60px`, then the reason why I left a bunch of `1px`s is because we don't need to worry about resizability at all, but I guess I could change them to rem it shouldn't matter that much because it's so small

Reply:
Fair... I just picked this one at random because I just saw the letters "px"... If it makes sense in this case, that's fine, and as Gary pointed, maybe I was taking the guidance too literally.